Sources/Rownd/Rownd.swift

```diff
@@ -180,9 +197,20 @@ public class Rownd: NSObject {
     }
     
     public static func requestSignIn(_ signInOptions: RowndSignInOptions?) {
+        var signInOptions = signInOptions
```

Comment:
Since we use this in multiple places (at least two), maybe it makes sense to move it to its own function that returns the validated `signInOptions` obj.

Reply:
Yup good idea! Fix has been added.